rhel8/profiles/stig.profile

```diff
@@ -21,6 +21,7 @@ extends: ospp
 selections:
     - login_banner_text=dod_banners
     - dconf_gnome_banner_enabled
+    - dconf_gnome_login_banner_text
```

Comment:
I noticed that in STIG profile, GNOME banners are enabled but the banner text is not configured in dconf.
So I'm selecting rule `dconf_gnome_login_banner_text`.

CC: @tedbrunell